src/aws/s3/index.ts

```diff
@@ -0,0 +1,25 @@
+import S3 from 'aws-sdk/clients/s3';
+import fs from 'fs';
+import config from '../../config/config.ts';
+
+export const { bucketName, region, accessKeyId, secretAccessKey } =
+  config.aws.s3;
+
+export const s3 = new S3({
+  region,
+  accessKeyId,
+  secretAccessKey,
+});
+
+//upload a file to s3
+export const uploadToS3 = (file: any) => {
```

Comment:
Instead of a function this could be a service that you inject via inversify or you can try to inject an instance of S3 class itself. Try to use DI more.